File: Podoal/app/src/main/java/kr/fanbridge/podoaltv/util/ProgressUtil.java
package kr.fanbridge.podoaltv.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import kr.fanbridge.podoaltv.R;


/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2023/Kpopcon/app/src/main/java/com/creator/kpopcon/util/ProgressUtil.kt
package com.creator.kpopcon.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.kpopcon.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}